lib/lure/mvn.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func mvnOutdated(path string) (error, []moduleVersion) {
-	cmd := exec.Command("mvn", "versions:display-dependency-updates", "-DprocessDependencyManagement=false")
+	cmd := exec.Command("mvn", "versions:display-dependency-updates", "-DprocessDependencyManagement=false", "-Dmaven.version.rules=file:Rules.xml")
```

Comment:
what happens if the file does not exist?

Reply:
Nothing. You get exactly the same output as if you don't have the parameter